doc/sdk-cli-reference.md

```diff
@@ -158,30 +162,53 @@ Create app-operator/.gitignore
 
 ## test
 
-### Flags
+### Available Commands
+
+#### local - Runs the tests locally
 
-* `-t, --test-location` **(required)** string - location of e2e test files
+##### Flags
 * `-k, --kubeconfig` string - location of kubeconfig for kubernetes cluster
 * `-g, --global-init` string - location of global resource manifest yaml file
 * `-n, --namespaced-init` string - location of namespaced resource manifest yaml file
```

Comment:
We need to revise the flags a bit to make them consistent. 
Let's get rid of short flags to avoid naming collisions.
And change:
- `namespaced-init` to `namespaced-manifest`
- `global-init` to `global-manifest`
- Keep `--namespace` as the flag for running the tests in a single namespace